plugins/jira/jira.plugin.zsh

```diff
@@ -59,6 +59,16 @@ open_jira_issue () {
   fi
 }
 
+jira_url_help() {
+  cat << EOF
+JIRA url is not specified anywhere.
+Valid options, in order of preference:
+  .jira-url file
+  $HOME/.jira-url file
```

Comment:
Good catch. I forgot to run it after I made that change, so I was expecting a literal "$HOME."